app/(auth)/verify-email/page.jsx

```
@@ -0,0 +1,138 @@
+                                Click để tải lên ảnh đại diện
+                            </span>
+                        </div>
+
+                        <div className="flex gap-4 mt-4">
+                            <button
+                                onClick={handleSkip}
+                                className="px-4 py-2 text-[--md-sys-color-primary]"
+                            >
+                                Bỏ qua
+                            </button>
+                            <button
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)